projects/kubernetes/autoscaler/Makefile

```diff
@@ -16,6 +18,16 @@ FIX_LICENSES=$(REPO)/cluster-autoscaler/LICENSE
 HAS_RELEASE_BRANCHES=true
 EXCLUDE_FROM_STAGING_BUILDSPEC=true
 
+HAS_HELM_CHART=true
+IMAGE_REPO=public.ecr.aws/a9u1e4h1
+
+HELM_SOURCE_OWNER=kubernetes
+HELM_SOURCE_REPOSITORY=autoscaler
+HELM_DIRECTORY=charts/cluster-autoscaler
+HELM_DESTINATION_REPOSITORY=autoscaler/charts/cluster-autoscaler
```

Comment:
Is this wired up correctly? I tried to follow what Ivy did [here](https://github.com/aws/eks-anywhere-build-tooling/blob/c4373b5c43b06fd32d8c9a24e5f89b59ca7f23c7/projects/aws-observability/aws-otel-collector/Makefile#L25-L26).